inst/include/Rcpp/api/meat/Rcpp_eval.h

```diff
@@ -54,7 +54,7 @@ inline SEXP Rcpp_eval(SEXP expr, SEXP env) {
     SET_TAG(CDDR(CDR(call)), ::Rf_install("interrupt"));
 
     // execute the call
-    Shield<SEXP> res(::Rf_eval(call, R_GlobalEnv));
+    Shield<SEXP> res(::Rf_eval(call, env));
```

Comment:
The `evalq` call itself might be unneeded, really -- we could probably omit constructing of the `evalq` bit and just evaluate the `tryCatch` expression in `env`.

We want to ensure that `expr` is evaluated in `env`, but the other usages likely want to just be evaluated in the global environment.